packages/@dcl/crdt/src/index.ts

```diff
@@ -1,38 +1,54 @@
-import { CRDT, Message, Payload, State } from './types'
-export * from './types'
+import { createGSet } from './gset'
+import {
+  ComponentDataMessage,
+  CRDT,
+  CRDTMessage,
+  CRDTMessageType,
+  DeleteEntityMessage,
+  Payload,
+  ProcessMessageResultType,
+  State
+} from './types'
 
 const globalBuffer = (globalThis as any).Buffer
 
+export * from './types'
+
 /**
  * Compare raw data.
  * @internal
+ * @returns 0 if is the same data, 1 if a > b, -1 if b > a
  */
-export function sameData<T>(a: T, b: T): boolean {
+export function dataCompare<T>(a: T, b: T): number {
   // At reference level
-  if (a === b) return true
+  if (a === b) return 0
+  if (a === null && b !== null) return -1
+  if (a !== null && b === null) return 1
 
   if (a instanceof Uint8Array && b instanceof Uint8Array) {
-    if (a.byteLength !== b.byteLength) {
-      return false
-    }
-
-    for (let i = 0; i < a.byteLength; i++) {
-      if (a[i] !== b[i]) {
-        return false
+    const n = a.byteLength > b.byteLength ? b.byteLength : a.byteLength
+    for (let i = 0; i < n; i++) {
+      const res = a[i] - b[i]
+      if (res !== 0) {
+        return Math.sign(res)
```

Comment:
For speed, prevent calling functions
```suggestion
        return res > 1 ? 1 : -1
```